src/util/hass-translation.js

```diff
@@ -36,24 +36,24 @@ export function getActiveTranslation() {
     if (translation) {
       return translation;
     }
-  } else if (navigator.languages) {
-    for (let i = 0; i < navigator.languages.length; i++) {
-      translation = languageGetTranslation(navigator.languages[i]);
+  }
+  if (navigator.languages) {
+    navigator.languages.forEach((locale) => {
+      translation = languageGetTranslation(locale);
       if (translation) {
         return translation;
```

Comment:
You'd better use for of as else the return will only return from the arrow function